src/olympia/promoted/tests/test_models.py

```diff
@@ -163,6 +163,66 @@ def test_get_resigned_version_number(self):
         assert addon.current_version is None
         assert promo.get_resigned_version_number() is None
 
+    def test_check_subscription_for_approval(self):
```

Comment:
I'll repeat my comment above and promote it to more than a nit. ;-)  I find these test cases really difficult to parse not knowing what `check_subscription_for_approval` is expected to return. I.e., when is it True and when is it False?